be/src/common/config.h

```diff
@@ -601,6 +604,12 @@ CONF_mInt32(remote_storage_read_buffer_mb, "16");
 // the level equal or lower than mem_tracker_level will show in web page
 CONF_Int16(mem_tracker_level, "0");
 
+// The minimum length when TCMalloc Hook consumes/releases MemTracker, consume size
+// smaller than this value will continue to accumulate. specified as number of bytes.
+// Decreasing this value will increase the frequency of consume/release.
+// Increasing this value will cause MemTracker statistics to be inaccurate.
+CONF_mInt32(mem_tracker_consume_min_size_mbytes, "1048576");
```

Comment:
1048576 mb ？？
And the default 4M is written in your document.
Which is the correct number ?

Reply:
1048576 bytes, 1M, see the last digit of the variable name and the comment.
The document is wrong, I will revise it, thx~

Reply:
Then your variable name should be called 'xxx_bytes' not 'xxx_mbytes'

Reply:
Thats right